src/main/java/org/sonarqube/gradle/SonarQubeTask.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.sonarqube.gradle;
 
+import java.util.HashMap;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.HashMap'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)